README.md

```diff
@@ -27,3 +27,4 @@ Java SE Major Release | End of Java SE Oracle Public Updates | Proposed End of S
 
 [1]: http://samtools.sourceforge.net
 [2]: http://vcftools.sourceforge.net/specs.html
+
```

Comment:
this was to trigger another check. i'm not waiting 1.5 hours for travis again to get rid of this